pkg/relic/os_util.sh

```diff
@@ -2,5 +2,5 @@ OSNAME=`uname -s`
 SEDBIN="sed -i"
 if [ "${OSNAME}" = "Darwin" ] ; then
     SEDBIN="sed -i ''"
-    LANG=C
```

Comment:
maybe leave this too, it was needed in the past -> maybe for some older version of macOS it should stay?

Reply:
Ok, will change.